### PR TITLE
abis: Define PACKET_HOST

### PR DESCRIPTION
**Pull request by Dennisbonke**
This PR adds a bunch of missing constants into the Linux ABI. It also stubs 2 functions and adds 2 new sysdeps.

Part of the mlibc LFS project.

### CODE REVIEW

**Comment by avdgrinten** [comment]
Huge PRs with lots of unrelated commits are quite annoying to review; it'd be better to group commits into semantically related changes.

**Comment by Dennisbonke** [comment]
> Huge PRs with lots of unrelated commits are quite annoying to review; it'd be better to group commits into semantically related changes.

If requested, I can do that. However, that will lead to a _lot_ of PRs as most commits aren't really related. I'll split out part 1, 2 and 3 shortly.

**Comment by avdgrinten** [comment]
Lots of PRs are not an issue. A small PR and be reviewed + merged in 5min, a long one takes weeks.

File: options/linux/include/netpacket/packet.h [hunk 1]
```diff
@@ -1,6 +1,8 @@
 #ifndef _NETPACKET_PACKET_H
 #define _NETPACKET_PACKET_H
 
+#define PACKET_HOST 0
```
Comment (64) [inline options/linux/include/netpacket/packet.h:4]: LGTM now that this PR is only a single line change :^)

**Review by 64 — APPROVED** [review]

File: options/linux/include/netpacket/packet.h (hunk 1 above)
Comment (ArsenArsen) [reply options/linux/include/netpacket/packet.h:4]: Isn't this an ABI thing?

Comment (Dennisbonke) [reply options/linux/include/netpacket/packet.h:4]: If it is, it matches Linux so it should be good.

Comment (64) [reply options/linux/include/netpacket/packet.h:4]: I think Arsen means that it should live in `abis/`

**Review by 64 — CHANGES_REQUESTED** [review]
As per the above comments, this should live in `abis/`.

**Review by Geertiebear — APPROVED** [review]
LGTM.

**Pull request merged**